src/wrappers/responsiveContainingView/component.tsx

```diff
@@ -78,6 +78,7 @@ export const ResponsiveContainingView = (props: IResponsiveContainingViewProps):
       size={{ base: props.size, ...props.sizeResponsive }}
       isFullWidth={isFullWidth}
       shouldIncludeMaxSize={shouldIncludeMaxSize}
+      centerHorizontally={props.centerHorizontally}
```

Comment:
dont pass as prop, pass as a class - it makes it more stable and less performance impact + easier to turn into static classes in the future